packages/gatsby/src/utils/webpack.config.js

```diff
@@ -509,7 +513,15 @@ module.exports = async (
       },
       minimizer: [
         // TODO: maybe this option should be noMinimize?
-        !program.noUglify && plugins.minifyJs(),
+        !program.noUglify &&
+          plugins.minifyJs(
+            program.profile && {
```

Comment:
if `program.profile` is not there, then this resolves to `plugins.minifyJs(false)`. I'm not sure that is actually problematic or not, but it is incorrect. I think we should probably do a ternary with an empty object to be correct.

Reply:
I did test this and it wasn't complaining, but I can change this easily enough to make it clearer